### DISCUSSION TITLE
Enable `HOMEBREW_NO_INSTALLED_DEPENDENTS_CHECK`

### DISCUSSION THREAD

**Discussion by smorimoto** [actions/virtual-environments#8986] [Image configuration]
I think auto-update is disabled now, is there any reason not to enable it? It's a little silly to just want to install a package and then have to wait almost 10 minutes to get the package update.

**Comment by erik-bershel** [comment]
Hello @smorimoto!

We update images as often as possible. And images are not updated after generation - it's a part of public contract. Unfortunately, updating images on the fly is guaranteed to lead to bigger problems than waiting for dependencies to be updated during runtime. If they are updated automatically, this will lead to a reduction in repetition. 
At the same time, we do not want to set a specific behaviour for users builds (disable dependency updating using the specified flag), since this often breaks than helps. At runtime, the user can set the desired flag and ignore updates if desired. There are examples of this behaviour in the repository. All related to broken dependencies updates: https://github.com/actions/runner-images/issues/7742, https://github.com/actions/runner-images/issues/8104.
